CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing guidelines
+
+## Contributing steps
+
+* Fork the desired repo, develop and test your code changes.
+* Submit a pull request.
+* The [repo owners](OWNERS) will respond to your PR.
+* Existing issues, feature requests are tracked in [Jira](https://jira.coreos.com/projects/CO/issues/).
+
+## How to build and run unit tests locally
+
+* Build and test
+
+  ```bash
+  make
+  ```
+
+* To only test
+
+  ```bash
+  make test
+  ```
```

Comment:
This doc feels a bit redundant with docs/developing.md, do you think we need a new document? Would it be better to make this a "Developing Quickstart" at the start of developing.md? I don't know how useful this is, if you're contributing it's not nearly enough to just run the unit tests, you need to know how to deploy and test the app itself.

Reply:
Maybe rename developing.md to contributing?

Reply:
Right, usually contributing document contains more than this. Hive is not ready for external contributors yet, so may be we can remove this. The quickstart guide should able to give initial on boarding information.

Reply:
The reason I added this because these are the files e.g. contributing, quickstart etc that usually new contributors look for.